yarn-project/end-to-end/src/e2e_pending_note_hashes_contract.test.ts

```diff
@@ -79,10 +79,10 @@ describe('e2e_pending_note_hashes_contract', () => {
         owner,
         deployedContract.methods.insert_note.selector,
         deployedContract.methods.get_then_nullify_note.selector,
-        deployedContract.methods.get_note_zero_balance.selector,
       )
       .send()
       .wait();
+    await expect(deployedContract.methods.get_note_zero_balance(owner).send().wait()).rejects.toThrow();
```

Comment:
I think it would be better to check the message here to ensure we don't throw for some other reason.